api/server/router/local/local.go

```diff
@@ -0,0 +1,161 @@
+package local
```

Comment:
PS: In the future, we could split that into separated routers, like a router for containers, or a router for images, but that's an exercise for later.